Update environments.md (#4143)
chef/chef-web-docs: content/environments.md

An environment can be created in four different ways:

> - Creating a Ruby file in the environments sub-directory of the

Author: ```suggestion
- Create a Ruby file in the environments sub-directory of the
```

>     chef-repo and then pushing it to Chef Infra Server

Author: ```suggestion
    chef-repo and then push it to Chef Infra Server
```

> - Creating a JSON file directly in the chef-repo and then pushing it

Author: ```suggestion
- Create a JSON file directly in the chef-repo and then push it
```
